Scripts/script-Base64ListToFile.yml

```diff
@@ -13,12 +15,17 @@ script: |-
 
   # get the list
   res = demisto.executeCommand('getList', {'listName': demisto.args()['listname']})
+  if res['Type'] == entryTypes['error']:
+    raise Exception("error reading list %s from demisto" % demisto.args()['listname'])
+    sys.exit(0)
 
   # convert base64 file to binary file
   bin_file = base64.decodestring(res[0]['Contents'])
+  if demisto.args()['isZipFile']:
```

Comment:
Did you mean to check if `yes` then get in the condition? The code in the condition will always be executed while any string is given to the argument.

Reply:
sure:)